addons/website_hr_recruitment/tests/test_website_hr_recruitment.py

```diff
@@ -12,10 +12,11 @@ def test_tour(self):
             'is_published': True,
         })
 
-        self.start_tour("/", 'website_hr_recruitment_tour')
+        with odoo.tests.RecordCapturer(self.env['hr.applicant'], []) as capt:
+            self.start_tour("/", 'website_hr_recruitment_tour')
 
         # check result
-        record = self.env['hr.applicant'].search([('description', '=', '### HR RECRUITMENT TEST DATA ###')])
+        record = capt.records
```

Comment:
```py
self.assertEqual(record.description, '### HR RECRUITMENT TEST DATA ###')
```

Reply:
Same as above, the description is `<p>### HR RECRUITMENT TEST DATA ###</p>`, should be without `<p>` but it is fine for now.